src/main/java/org/mule/extension/aggregator/internal/privileged/executor/AbstractAggregatorExecutor.java

```diff
@@ -213,6 +222,30 @@ public void start() throws MuleException {
     }
   }
 
+  private void executeDeferActions() {
+    quorumManager = new Thread(() -> {
+      boolean startCompleted = false;
+      while (!startCompleted) {
+        try {
+          Thread.sleep(200);
```

Comment:
Please check the sleep function parameter with @lucianoRM  (I don't know if we should use a fixed value or a default value and create a system property to allow setting it)